### ISSUE TITLE
Training only Human class, map with Pedestrian class in Kitti but result is 0.0000

### ISSUE THREAD

**Issue by thanhpt93** [open-mmlab/OpenLidarPerceptron#1698]
Hi everyone,
I have issue that when I train only class is human, and I mapping with pedestrian class in Kitti. But result is not as expect, all is 0.0000 as the bellow:

2025-01-05 10:33:55,387   INFO  *************** Performance of EPOCH 10 *****************
2025-01-05 10:33:55,387   INFO  Generate label finished(sec_per_example: 0.6781 second).
2025-01-05 10:33:55,387   INFO  recall_roi_0.5: 0.775510
2025-01-05 10:33:55,387   INFO  recall_rcnn_0.5: 0.843537
2025-01-05 10:33:55,387   INFO  recall_roi_0.7: 0.442177
2025-01-05 10:33:55,387   INFO  recall_rcnn_0.7: 0.455782
2025-01-05 10:33:55,387   INFO  Average predicted number of objects(78 samples): 1.962
/home/thanhpt/miniconda3/envs/3d/lib/python3.8/site-packages/numba/cuda/dispatcher.py:536: NumbaPerformanceWarning: Grid size 9 will likely result in GPU under-utilization due to low occupancy.
  warn(NumbaPerformanceWarning(msg))
2025-01-05 10:33:55,414   INFO  Pedestrian AP@0.50, 0.50, 0.50:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00
Pedestrian AP_R40@0.50, 0.50, 0.50:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00
Pedestrian AP@0.50, 0.25, 0.25:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00
Pedestrian AP_R40@0.50, 0.25, 0.25:
bbox AP:0.0000, 0.0000, 0.0000
bev  AP:0.0000, 0.0000, 0.0000
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:0.00, 0.00, 0.00

For more detail, I have pretrain is PV-RCNN. VOXEL SIZE is (0.1, 0.1, 0.1). Custom data has been checked annotation is correct
I wonder if anyone has encountered this. Hope to hear from you guys!

**Comment by Petros626** [comment]
maybe from Pedestrians there aren't so many samples contained? What is the number Pedestrians for `gt_sampling` in the `.yaml`? How does your training loss look like?
